tests/core.py

```diff
@@ -2185,8 +2185,8 @@ def test_init_proxy_user(self):
         self.assertEqual('someone', c.proxy_user)
 
 
-HDFSHook = None
-snakebite = None
+HDFSHook: None = None
+snakebite: None = None
```

Comment:
This looks like invalid annotations.

Reply:
This part of the code kind of confuses me. Looks like they are never set to other value after being initialized as `None`?